assets/script/main.js

```diff
@@ -0,0 +1,75 @@
+const myLibrary = [];
+
+// Book constructor
+function Book() {
+  this.title = document.getElementById('title').value;
+  this.author = document.getElementById('author').value;
+  this.pages = document.getElementById('pages').value;
+  this.read = (function () {
+    if (document.getElementById('yes').checked) return document.getElementById('yes').value;
+    return document.getElementById('no').value;
+  }());
+}
+
+// Displaying book to the UI
+function displayBook(book) {
+  const list = document.querySelector('#book-list');
+  const row = document.createElement('tr');
+
+  row.innerHTML = `
+    <td class="mx-4">${book.title}</td>
+    <td class="mx-4">${book.author}</td>
+    <td class="mx-4">${book.pages}</td>
+    <td><a href="#" class="btn btn-primary mx-4" id="statusYes">${book.read}</a></td>
+    <td><a href="#" class="btn btn-danger mx-4 delete">Remove</a></td>
+    `;
+  list.appendChild(row);
+}
```

Comment:
- display book function should loop through all the books for the librarybooks variable and display them one by one.
![image](https://user-images.githubusercontent.com/34653764/105484034-05b76480-5cab-11eb-9886-6e775984b95c.png)